app/main/views/services.py

```diff
@@ -181,31 +157,26 @@ def import_service(service_id):
     """
     is_valid_service_id_or_400(service_id)
 
-    now = datetime.now()
+    service = Service.query.filter(
+        Service.service_id == service_id
+    ).first()
 
-    json_payload = get_json_from_request()
-    json_has_required_keys(json_payload,
-                           ['services', 'update_details'])
+    if service is not None:
+        abort(400, "Cannot update service by PUT")
```

Comment:
Should the message rather be that you can't PUT to an ID that already exists? (I know this is just what was there already)